cmake/external/googletest.cmake

```diff
@@ -13,18 +13,14 @@
 # limitations under the License.
 
 include(ExternalProject)
-include(ExternalProjectFlags)
-
-ExternalProject_GitSource(
-  GOOGLETEST_GIT
-  GIT_REPOSITORY "https://github.com/google/googletest.git"
-  GIT_TAG "release-1.8.0"
-)
 
 ExternalProject_Add(
   googletest
 
-  ${GOOGLETEST_GIT}
+  DOWNLOAD_DIR ${PROJECT_BINARY_DIR}/downloads
+  DOWNLOAD_NAME googletest-1.8.0.tar.gz
+  URL https://github.com/google/googletest/archive/release-1.8.0.tar.gz
+  URL_HASH MD5=16877098823401d1bf2ed7891d7dce36
```

Comment:
Where are you getting these MD5s from? If you're computing them by hand prefer something stronger, like SHA-256. On a linux and mac, compute with `shasum -a 256 filename`

Reply:
Done.